pkg/monitoring/rules/alerts/operator.go

```diff
@@ -58,16 +58,28 @@ var operatorAlerts = []promv1.Rule{
 	},
 	{
 		Alert: "CDIDataImportCronOutdated",
-		Expr:  intstr.FromString(`sum by(ns,cron_name) (kubevirt_cdi_dataimportcron_outdated{pending="false"}) > 0`),
+		Expr:  intstr.FromString(`sum by(namespace,cron_name) (kubevirt_cdi_dataimportcron_outdated{pending="false", namespace=~"openshift-virtualization-os-images|kubevirt-os-images"}) > 0`),
```

Comment:
So I am struggling a bit with CDI having to "know" about "golden image namespaces" or "special dataimportcrons".
I think we should either label the metric according to labels on the cron that indicate it's "managed" or move this definition to other operators that encapsulate this knowledge.

Alternatively, we could also label the metric with this information though I don't know if anything gives it away easily.

Reply:
From what I was told the images in openshift-virtualization-os-images|kubevirt-os-images are the golden images.